math-scala/src/main/scala/org/apache/mahout/drivers/MahoutOptionParser.scala

```diff
@@ -50,7 +50,7 @@ class MahoutOptionParser(programName: String) extends OptionParser[Map[String, A
         "(optional). Default: empty.")
     }
 
-    opt[String]('o', "output") required() action { (x, options) =>
+    opt[String]('o', "output") action { (x, options) =>
```

Comment:
or just don't call that helper and put your own output option in your driver. I made these helpers to keep some options standardized but no hard rule.